.github/workflows/ci.yaml

```
@@ -22,7 +22,8 @@ jobs:
 
   bazel:
     runs-on: ubuntu-latest
-    container: gcr.io/gapic-images/googleapis-bazel:20210105
+    # Do we need to use gcr.io/gapic-images/googleapis-bazel:20210105 ?
+    container: gcr.io/cloud-devrel-kokoro-resources/java8
```

you will likely need to use the public version of the image so that github can pull it
```suggestion
    container: gcr.io/cloud-devrel-public-resources/java8
```

---

Thank you!